<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewRemark</class>
 <widget class="QDialog" name="viewRemark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IGN RIPart : Historique de la remarque</string>
  </property>
  <property name="windowIcon">
   <iconset theme="IGN" resource="resources.qrc">
    <normaloff>:/plugins/RipartPlugin/images/ign.png</normaloff>:/plugins/RipartPlugin/images/ign.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Statut</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Message de la remarque</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblOldResponse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Réponse(s) antérieure(s)</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textOldRep">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="textStatut">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>statut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCroquis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sélection/désélection des croquis associés à la remarque&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Croquis</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/plugins/RipartPlugin/images/magicwand.png</normaloff>:/plugins/RipartPlugin/images/magicwand.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblCntCroquis">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de croquis sélectionnés: -</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditCntCroquisDetail">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>accepted()</signal>
   <receiver>viewRemark</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>rejected()</signal>
   <receiver>viewRemark</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
